server/controllers/profile.js

```diff
@@ -0,0 +1,61 @@
+const express = require('express');
+const Profile = require('../models/Profile');
+const asyncHandler = require("express-async-handler");
+
+// @route POST /profile/create
+// @desc Create profile
+// @access Public
+exports.createProfile = asyncHandler(async (req, res, next) => {
+    const profile = new Profile(req.body);
```

Comment:
It's a good security practice to destructure and validate user inputs, rather than pass all directly to your DB.